lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -237,11 +238,17 @@ def race(races: str or list) -> list:
 
     race_map = {
         "American Indian or Alaska Native": "americanIndianOrAlaskaNative",
+        "amerind": "americanIndianOrAlaskaNative",
         "Asian": "asian",
+        "asian": "asian",
         "Black or African American": "blackOrAfricanAmerican",
+        "black": "black",
```

Comment:
Should the value here be "blackOrAfricanAmerican" to match others?

Reply:
Yes thank you